Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -210,9 +211,10 @@ def self.check_http_content(url, user_agents: [:default])
     "The URL #{url} could use HTTPS rather than HTTP"
   end
 
-  def self.http_content_headers_and_checksum(url, user_agent: :default)
+  def self.http_content_headers_and_checksum(url, hash_needed = false, user_agent: :default)
```

Comment:
You'll need to use `hash_needed: false` in both the method definition and the callers of that method, too. Passing boolean arguments unless it's an accessor is generally not great because it's not as readable.